site/docs/pages/is-entrypoint.mdx

```diff
@@ -0,0 +1,34 @@
+# `isValidAAEntrypoint`
+
+The `isValidAAEntrypoint` utility is designed to verify if the entrypoint address is a valid v6 entrypoint.
```

Comment:
I think we explain a bit more here what's going.

Like what kind of address we are checking. And where this method should be used.

Reply:
Updated to:

```
The `isValidAAEntrypoint` utility is designed to verify if the target account abstraction singleton address.
This should be used to validate the entrypoint before sponsoring a transaction.
```